### ISSUE TITLE
Working code of Optical Character Recognition (OCR)

### ISSUE THREAD

**Issue by Gaurav7888** [adiXcodr/node-ocr#7]

**Comment by imsiddhant07** [comment]
Hello, 
If the issue is not assigned to anyone, may I work on it?

**Comment by Gaurav7888** [comment]
Hi , I opened this issue because i think its missing from this project if assigned then i want to work on this issue@adiXcodr

**Comment by imsiddhant07** [comment]
Yes true, but you should have provided a description while raising an issue and also asked about assigning it. Never the less let @adiXcodr decide :)

**Comment by Gaurav7888** [comment]
Ocr will take time and this is something  2 ppl can contribute  together , thats what i think @siddhantjain07 .

**Comment by adiXcodr** [comment]
Currently, the OCR is made possible with the Tesseract.js framework, if you guys want, you can write a native alternative algorithm but it would need training data

**Comment by Gaurav7888** [comment]
@adiXcodr can we make ocr code using tensorflow. Js , that will be helpful for ur project ?

**Comment by adiXcodr** [comment]
Yes that was what I had in mind too. If you want to, you can proceed with it

**Comment by harshil15999** [comment]
@adiXcodr does the OCR module have to be in javascript?

**Comment by Gaurav7888** [comment]
@adiXcodr i will start  working on it using tf.js ...

**Comment by adiXcodr** [comment]
> @adiXcodr does the OCR module have to be in javascript?

Preferably. Doing it in some other language like python would need an extra API call. The computation needs to be done natively
